pkgs/os-specific/linux/procps-ng/default.nix

```diff
@@ -1,15 +1,25 @@
-{ lib, stdenv, fetchurl, ncurses }:
+{ lib, stdenv, fetchFromGitLab, ncurses, libtool, gettext, autoconf, automake, pkgconfig }:
 
 stdenv.mkDerivation rec {
   name = "procps-${version}";
   version = "3.3.12";
 
-  src = fetchurl {
-    url = "mirror://sourceforge/procps-ng/procps-ng-${version}.tar.xz";
-    sha256 = "1m57w6jmry84njd5sgk5afycbglql0al80grx027kwqqcfw5mmkf";
+  src = fetchFromGitLab {
+    owner ="procps-ng";
+    repo = "procps";
+    # should be tag v3.3.12, but the sha is immutable of course, which is desirable
+    rev = "e0784ddaed30d095bb1d9a8ad6b5a23d10a212c4";
```

Comment:
I'd still use
```
rev = "v${version}";
```

Git tags are meant not to be moved; we check the hash of the output anyway.  It will also be less error-prone on updates.

Reply:
ok, will add that